src/filestore.rs

```diff
@@ -0,0 +1,224 @@
+use std::{
+    collections::BTreeMap,
+    fs,
+    path::{Path, PathBuf},
+};
+
+use colored::Colorize;
+use serde::Deserialize;
+
+use crate::{
+    model::{Annotation, GPUSpecRequirement, KeyInfo, Pod, ToYaml},
+    store::Store,
+};
+
+static POD_FOLDER_NAME: &str = "Pod";
+
+pub struct FileStore {
+    storage_folder_path: PathBuf,
+}
+
+impl FileStore {
+    pub fn new(storage_folder_path: PathBuf) -> Self {
+        Self {
+            storage_folder_path: storage_folder_path,
+        }
+    }
+
+    fn construct_annotation_path(&self, hash: &str) -> PathBuf {
+        let mut path_buf = self.storage_folder_path.to_path_buf();
+        path_buf.push("Annotation");
+        path_buf.push(format!("{}{}", hash, ".yaml"));
+
+        path_buf
+    }
+
+    fn construct_folder_path(&self, model_name: &str, hash: &str) -> PathBuf {
+        let mut path_buf = self.storage_folder_path.to_path_buf();
+        path_buf.push(model_name);
+        path_buf.push(format!("{}.yaml", hash));
+
+        path_buf
+    }
+}
+
+impl Store for FileStore {
+    fn store_annotation(
+        &self,
+        annotation: &crate::model::Annotation,
```

Comment:
please add `use` of Annotation for consistency and avoid fully qualified name

Reply:
Ah that was stupid vs code auto doing that